documentation/writing-installers.md

```diff
@@ -0,0 +1,100 @@
+# Writing Installers
```

Comment:
Lets call this guide "writing installers & generators".

The logic is essentially the same, the only thing is that if your task is called `<your_library>.install` then it will be invoked when `mix igniter.install` is called.